{"name": "test_companyRegister", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message:", "trace": "@pytest.mark.run(order=1)\n    @allure.severity(\"blocker\")\n    @allure.description(\"ÊµãËØï http://10.10.128.152:10053/user/register ‰∏≠Â∞èÂæÆ‰ºÅ‰∏öÊ≥®ÂÜåÊµÅÁ®ã\")\n    @allure.testcase(\"http://10.10.128.152:10053/user/register\", \"Ê≥®ÂÜå üëá\")\n    def test_companyRegister():\n>       companyPassForget(act)\n\ntest_flow\\test_Authentication\\test_login.py:53: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\ntest_flow\\test_Authentication\\test_login.py:189: in companyPassForget\n    waitUntilClick(driver, loginOn.btn_nextPwd1_css.value)\ncommon\\BaseFunction.py:29: in waitUntilClick\n    WebDriverWait(driver, 30).until(EC.element_to_be_clickable((By.CSS_SELECTOR, ByCss)))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"f2f31047557a30a35d4eb07b634b6f7e\")>\nmethod = <selenium.webdriver.support.expected_conditions.element_to_be_clickable object at 0x0000016E958748D0>\nmessage = ''\n\n    def until(self, method, message=''):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value is not False.\"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.time() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, 'screen', None)\n                stacktrace = getattr(exc, 'stacktrace', None)\n            time.sleep(self._poll)\n            if time.time() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\n..\\..\\..\\AppData\\Local\\Programs\\Python\\Python36\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py:80: TimeoutException"}, "description": "ÊµãËØï http://10.10.128.152:10053/user/register ‰∏≠Â∞èÂæÆ‰ºÅ‰∏öÊ≥®ÂÜåÊµÅÁ®ã", "attachments": [{"name": "log", "source": "9c2ecee9-a127-42fb-89e4-4e675ece8988-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "e60ea386-655d-42c5-8cdd-f65b07d7bb8b-attachment.txt", "type": "text/plain"}], "start": 1602495299346, "stop": 1602495340894, "uuid": "0cde6b82-1258-4293-aa75-2150c7aabbe3", "historyId": "dd0646a5c1a798e47c4f25747509dea9", "testCaseId": "1fe2bac8daa9cfa6b5ae5069ce9a9205", "fullName": "test_flow.test_Authentication.test_login#test_companyRegister", "labels": [{"name": "severity", "value": "blocker"}, {"name": "tag", "value": "run(order=1)"}, {"name": "parentSuite", "value": "test_flow.test_Authentication"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "A02-03-00033"}, {"name": "thread", "value": "6636-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_flow.test_Authentication.test_login"}], "links": [{"type": "test_case", "url": "http://10.10.128.152:10053/user/register", "name": "Ê≥®ÂÜå üëá"}]}